src/fillTank.js

```diff
@@ -37,3 +37,4 @@ function roundPrice(price) {
 }
 
 module.exports = { fillTank };
```

Comment:
The 'fillTank' function is being exported, but it is not defined in the provided code snippet. Make sure to define the function before exporting it.